src/screens/ProfileScreen/index.js

```diff
@@ -71,6 +71,7 @@ import {withInteractionsManaged} from '../../components/WithInteractionManaged';
 import ShadowFloatingButtons from '../../components/Button/ShadowFloatingButtons';
 import useCoreFeed from '../FeedScreen/hooks/useCoreFeed';
 import StorageUtils from '../../utils/storage';
+import ImageCompressionUtils from '../../utils/image/compress';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `ImageCompressionUtils` is a good addition as it will be used for image compression in the following code.
```diff
+ import ImageCompressionUtils from '../../utils/image/compress';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->